src/ngraph/runtime/cpu/builder/batch_norm.cpp

```diff
@@ -324,7 +325,17 @@ namespace ngraph
                 functors.emplace_back(functor);
             }
 
+            template <>
+            void Builder::BUILDER_DECL(ngraph::op::BatchNormRelu)
+            {
+                if (!mkldnn_utils::use_mkldnn_kernel(node))
+                {
+                    throw ngraph_error("BatchNormRelu is only supported with 4-D MKLDNN kernel.");
+                }
+                build_batch_norm(external_function, node, args, out, true);
```

Comment:
you will need to templatize build_batch_norm

Reply:
jayaram, isn't it by flipping relu  in `build_batch_norm ` flag we can control whether to execute Reluas post op, still do we need to tempaltize it?

Reply:
There is a static_cast to op type in build_batch_norm

Reply:
ah thanks for pointing , i overlooked that part :). will fix it.